sentry_sdk/client.py

```diff
@@ -417,6 +417,12 @@ def _capture_envelope(envelope):
 
             if self.options.get("spotlight"):
                 self.spotlight = setup_spotlight(self.options)
+                if not self.options["dsn"]:
+                    sample_all = lambda *_args: 1.0
```

Comment:
Let's add a `**_kwargs`, just in case. Not sure we ever call it that way, but can't hurt

```suggestion
                    sample_all = lambda *_args, **_kwargs: 1.0
```